wallet/src/main/java/com/android/identity_credential/wallet/PresentationActivity.kt

```diff
@@ -281,13 +290,17 @@ class PresentationActivity : FragmentActivity() {
         super.onCreate(savedInstanceState)
 
         setContent {
-            IdentityCredentialTheme {
+            PresentmentTheme {
```

Comment:
Why do we need a new theme here? I'm pretty sure the existing theme can do what we need, in particular I think you can use `MaterialTheme.colorScheme.surfaceContainerHigh` for the background....